src/orquestra/quantum/circuits/_matrices.py

```diff
@@ -221,6 +241,18 @@ def xy_matrix(angle):
     )
 
 
+def ms_matrix(phi_0, phi_1):
```

Comment:
It would be good to include the name of Mølmer-Sørenson somewhere?
Cause right now it's not obvious what MS means, and it's not even searchable.

Also in general it would be good to have some docs for these.

Reply:
Well there are docs for this gate in the link under this function. But I'll be happy to add in some more documentation.